packages/base-contract/src/index.ts

```diff
@@ -150,6 +159,38 @@ export class BaseContract {
         }
         return rawEncoded;
     }
+    public async evmExecAsync(input: Buffer): Promise<string> {
+        // should only run once, the first time it is called
+        if (this._evmIfExists === undefined) {
+            const vm = new VM();
```

Comment:
i wonder if it would be worthwhile to move the one-time code into a method, which could optionally be called by the client in order to quicken the first actual contract call.

Reply:
I think it'll be alright this way - any functions using the local EVM won't be time sensitive.